hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/handlers/ConsensusGetTopicInfoHandlerTest.java

```diff
@@ -74,6 +75,7 @@ void setUp() {
     }
 
     @Test
+    @DisplayName("extractHeader as expected")
```

Comment:
There's no value added with this and a few other of the display names. It's just extra code to read that duplicates what's already in the method name

Reply:
Just to show which method is tested.

Reply:
By default, junit already displays the method name. I would suggest either putting more info about the test in these messages, or just removing it

Reply:
I updated the display. Please see if it makes more sense to you. I tried to make sure every unit test method has @DisplayName.

Reply:
These look better, thank you! 😄